### PR TITLE
Handle NoLogAbsDetJacobian in ComposedFunction

### PR DESCRIPTION
**Pull request by oschulz**

### CODE REVIEW

File: src/with_ladj.jl [hunk 1]
```diff
@@ -86,9 +86,14 @@ with_logabsdet_jacobian(::F, ::T) where {F,T} = NoLogAbsDetJacobian{F,T}()
...
-        y_inner, ladj_inner = with_logabsdet_jacobian(f.inner, x)
-        y, ladj_outer = with_logabsdet_jacobian(f.outer, y_inner)
-        (y, ladj_inner + ladj_outer)
+        y_ladj_inner = with_logabsdet_jacobian(f.inner, x)
+        if y_ladj_inner isa NoLogAbsDetJacobian
+            NoLogAbsDetJacobian{typeof(f),typeof(x)}()
+        else
+            y_inner, ladj_inner = y_ladj_inner
+            y, ladj_outer = with_logabsdet_jacobian(f.outer, y_inner)
+            (y, ladj_inner + ladj_outer)
```
Comment (devmotion) [inline src/with_ladj.jl:95]: I think here one should also check if `with_logabsdet_jacobian` is a `NoLogAbsDetJacobian`? It seems the tests don't cover this case currently.

Comment (oschulz) [reply src/with_ladj.jl:95]: Oh, yes!

Comment (oschulz) [reply src/with_ladj.jl]: Fixed.

**Review by devmotion — APPROVED** [review]
LGTM.

**Comment by oschulz** [comment]
> LGTM.

Thanks! I'll tag a bugfix release.

**Pull request merged**